### PR TITLE
feat(accident-notification): An/status of notification

### PR DESCRIPTION
**Pull request by Thorrik58**
# Add status of notification

Attach a link to issue if relevant

## What

Add call to status of notification and fixed some bugs along the way.

## Why

Needed for notifying the user of the status of the application

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/api/domains/health-insurance/src/lib/graphql/models/accidentNotificationStatus.model.ts [hunk 1]
```diff
@@ -9,13 +9,13 @@ export class AccidentNotificationStatus {
   @Field(() => Number)
   numberIHI?: number
 
-  @Field(() => HealthInsuranceAccidentNotificationStatusTypes)
+  @Field(() => String)
   @IsEnum(HealthInsuranceAccidentNotificationStatusTypes)
-  status?: HealthInsuranceAccidentNotificationStatusTypes
+  status?: string
```
Comment (obmagnusson) [inline libs/api/domains/health-insurance/src/lib/graphql/models/accidentNotificationStatus.model.ts:14]: Change back to enum

Comment (Thorrik58) [reply libs/api/domains/health-insurance/src/lib/graphql/models/accidentNotificationStatus.model.ts]: fixed

File: libs/api/domains/health-insurance/src/lib/accident-notification.service.ts [hunk 2]
```diff
@@ -16,39 +16,39 @@ import {
...
+      return AttachmentTypes.INJURY_CERTIFICATE
     case 2:
-      return HealthInsuranceAccidentNotificationAttachmentTypes.PROXY_DOCUMENT
+      return AttachmentTypes.PROXY_DOCUMENT
     case 3:
-      return HealthInsuranceAccidentNotificationAttachmentTypes.POLICE_REPORT
+      return AttachmentTypes.POLICE_REPORT
     default:
-      break
+      return 'Unknown'
```
Comment (saevarma) [inline libs/api/domains/health-insurance/src/lib/accident-notification.service.ts:40]: Should `Unkown` also be on the AttachmentTypes enum?

**Review by saevarma — APPROVED** [review]
LGTM

**Pull request merged**